src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Windows.cs

```diff
@@ -10,9 +10,11 @@ namespace System.Threading
 {
     public static partial class ThreadPool
     {
-        internal static bool UseWindowsThreadPool { get; } =
+        private static readonly bool s_useWindowsThreadPool = // name relied on by sos
```

Comment:
Maybe move the comment above the field, it's a bit odd to see a `//` comment in the middle of a statement